modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/consensus/ConsensusManager.scala

```diff
@@ -77,7 +81,16 @@ object ConsensusManager {
           def getObservedOutcome: F[ConsensusOutcome[Key, Artifact]] = for {
             readyPeers <- clusterStorage.getResponsivePeers
               .map(_.filter(_.state === Ready))
-            selectedPeer <- Random[F].elementOf(readyPeers)
+            numberOfPeersToQuery <- Random[F].nextIntBounded(readyPeers.size / 4 + 1)
```

Comment:
What's that magic calculation? Every magic number or calculation should be named

Reply:
There is no magic calculation. We want to sample a small subset of peers. So, I'm taking at most 25% of the ready `Peer`s and at least 1 ready `Peer`. There is no rhyme or reason for the 25% number. It just seemed like a good number.

Reply:
First search from stackoverflow: https://stackoverflow.com/a/47890

Reply:
The process of sub-selecting the ready peers has been moved to and encapsulated in a function.